spec/factories/levelling_up_premium_payments/eligibilities.rb

```diff
@@ -9,6 +9,7 @@
       qualification { :postgraduate_itt }
       eligible_itt_subject { :mathematics }
       teaching_subject_now { true }
+      eligible_degree_subject { true }
```

Comment:
Is this required? It shouldn't need `eligible_degree_subject` if `eligible_itt_subject` is `:mathematics`.

Reply:
I was getting spec failures before this change but that seems to no longer be an issue after removing this so I will remove it now, thanks for flagging.

Reply:
Could it be related to removing `:submit` from the validation?